Makefile

```diff
@@ -73,11 +77,8 @@ docker-push: docker-build
 	docker push $(IMAGE_NAME):$(IMAGE_VERSION)
 
 clean:
-	rm -rf .tmp
-	rm -rf phase3/.tmp
-	rm -rf phase1/gce/.tmp
-	rm -rf phase1/azure/.tmp
-	rm -rf phase1/vsphere/.tmp
+	rm -rf phase3/*/.tmp/
+	rm -rf phase1/*/*/
```

Comment:
This phase1 rm seems scary to me, as someone who routinely makes local directories in git repos to hold temporary data, since this would match any directory that wasn't even necessarily created as a temp directory for jsonnet/terraform purposes. Can this be much more explicit about what temp directories to delete?

I personally am fine if it just uses the current cluster name, meaning that `make clean` would only clean up after the current cluster instead of all possible clusters.